v1_20/src/main/java/com/denizenscript/denizen/nms/v1_20/helpers/EntityHelperImpl.java

```diff
@@ -767,16 +760,6 @@ public void setUUID(Entity entity, UUID id) {
         }
     }
 
-    @Override
-    public float getStepHeight(Entity entity) {

```

Comment:
Removed but not yet replaced?

Reply:
Same as some other features - only broken on 1.20.6 (older version's NMS impl still exists) and added a TODO since the mech can be deprecated & backsupported with the attribute